packages/transition-backend/file/__tests__/definitions.test.ts

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs';
+
+// Language configuration - TODO: this should be moved to a separate config file
+const languages = [
+    'en',
+    'fr'
+];
+
+describe('LaTeX Label Consistency Tests', () => {
+    // Helper function to extract labels from LaTeX content
+    function extractLabels(content) {
+        const labelRegex = /\\label{([^}]+)}/g;
```

Comment:
Give a short description of what the regex does (specifically the part after 'label')

Reply:
added